accounts/forms.py

```diff
@@ -127,15 +127,13 @@ def get_user_by_email(email):
 class UsernameField(forms.CharField):
     """ Username field, 3~30 characters, allows only alphanumeric chars, required by default """
     def __init__(self, required=True):
-        required_str = _("Required. ") if required else ""
-
         super(UsernameField, self).__init__(
             label=_("Username"),
             min_length=3,
             max_length=30,
             validators=[RegexValidator(r'^[\w.+-]+$')],  # is the same as Django UsernameValidator except for '@' symbol
-            help_text=_(required_str + _("30 characters or fewer. Can contain: letters, digits, underscores, dots, "
-                                         "dashes and plus signs.")),
+            help_text=_(_("30 characters or fewer. Can contain: letters, digits, underscores, dots, "
```

Comment:
the double underscore here seems weird...

Reply:
Fixed in https://github.com/MTG/freesound/pull/1177/commits/b53628dc0101b4718be271805e9133151242844c